### DISCUSSION TITLE
Welcome to ReasSonus FaderPort Discussions!

### DISCUSSION THREAD

**Discussion by navelpluisje** [navelpluisje/reasonus-install#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wavemechanix** [comment]
Hi, am loving Reasonus so far. Bought a Faderport 16 hoping I could get it to work with Reaper, and your work is the solution so thanks so much. All of my work is done in Reaper, the FP would be very limited without Reasonus.
I'd like to buy you a coffee but can't find the link, where is it?

**Reply by navelpluisje** [reply]
Hi [wavemechanix](https://github.com/wavemechanix), Thanx for the kind words. The link for BuyMeACofee is https://www.buymeacoffee.com/navelpluisje

**Comment by wavemechanix** [comment]
Hey the link doesn't seem to work:
[image: image.png]

**Reply by navelpluisje** [reply]
Hmm, that's weird, I can just click it and get there. Image is not in the post.

**Comment by sonicbreeze** [comment]
You have made the purchase of Faderport 2 possible. It such a great tool. I did post a bug I was having with the Touch automation which you will see in the "bug report" area and hopefully you can address that for me.
But I have a request for a feature too for my Faderport 2: It would be great when multiple tracks are selected, to have the ability to toggle Solo or Mute on all the selected tracks, and also to control all their levels with the fader. Could feature be added?

**Comment by sagreen83** [comment]
I have a presonus ioStation24c (faderport2 with 2 mic pres). It doesnt seem to be recognized by the install process. Is there a way to get the ioStation working with reasonus? If support needs to be added, I'm happy to make the changes to the code, I would just need a few hints as to what needs to be changed.
